src/parser.lua.j2

```diff
@@ -0,0 +1,4 @@
+project = {{ project | to_lua }}
```

Comment:
Actually, since we're not using luadata, I don't see any good reason to use this file. It might be better to just delete it and add these lines to the top of `script.lua.j2` instead.